src/docs/user/references/Variables.adoc

```diff
@@ -226,12 +242,45 @@ Note that #value always contains a string and must be converted to other types i
 
 NOTE: the SpEL expression must return a boolean value, this is why in the above expressions we use the `t(expression)` function to perform affectations and return a boolean `true` value.
 
+[[_dynamic_forms]]
+====== Dynamic form example using SpEL
+Using SpEL expressions, it is possible to show or hide variables based on the values of other variables. Thus, it allows to create _dynamic forms_.
 
-[[_dynamic_variables]]
+Consider the following variables definition:
+
+```xml
+  <variables>
+    <variable name="type" value="vegetable" model="PA:LIST(vegetable,fruit)" description="" group="" advanced="false" hidden="false"/>
+    <variable name="potatoes" value="0" model="PA:INTEGER" description="Amount of potatoes to order (in kilograms)" group="vegetables" advanced="false" hidden="false"/>
+    <variable name="leek" value="0" model="PA:INTEGER" description="Amount of leek to order (in kilograms)" group="vegetables" advanced="false" hidden="false"/>
+    <variable name="apples" value="0" model="PA:INTEGER" description="Amount of apples to order (in kilograms)" group="fruits" advanced="false" hidden="true"/>
+    <variable name="oranges" value="0" model="PA:INTEGER" description="Amount of oranges to order (in kilograms)" group="fruits" advanced="false" hidden="true"/>
+    <variable name="type_handler" value="" model="PA:SPEL(variables[&#39;type&#39;] == &#39;vegetable&#39; ? showGroup(&#39;vegetables&#39;) &amp;&amp; hideGroup(&#39;fruits&#39;) : showGroup(&#39;fruits&#39;) &amp;&amp; hideGroup(&#39;vegetables&#39;))" description="" group="" advanced="false" hidden="true"/>
```

Comment:
Replace `&#39;` back to quote could be more clear for the document.

Reply:
ok